piqe_ocp_lib/api/resources/ocp_node_metrics.py

```diff
@@ -0,0 +1,50 @@
+from piqe_ocp_lib.api.resources.ocp_base import OcpBase
+from kubernetes.client.rest import ApiException
+import logging
+from piqe_ocp_lib import __loggername__
+
+logger = logging.getLogger(__loggername__)
+
+
+class OcpNodeMetrics(OcpBase):
+    """
+    OcpNodeMetrics Class extends OcpBase and encapsulates all methods
+    related to managing Openshift nodes Metrics.
+    :param kube_config_file: A kubernetes config file.
+    :return: None
+    """
+
+    def __init__(self, kube_config_file=None):
+        super(OcpNodeMetrics, self).__init__(kube_config_file=kube_config_file)
+        self.api_version = 'metrics.k8s.io/v1beta1'
+        self.kind = 'NodeMetrics'
+        self.ocp_nodes = self.dyn_client.resources.get(api_version=self.api_version, kind=self.kind)
+
+    def get_a_node_metrics(self, node_name):
```

Comment:
I would suggest changing this method name to get_node_metrics_name or something like that I'm finding it a little confusing have this name get_a_node_metrics and the other method being named get_node_metrics

Reply:
I gave this names which is what we have used in ocp_nodes.py file for consistency. For single node vs all nodes. Please let me know what do you think?

Reply:
Updated the method names.